exchange/utils.go

```diff
@@ -118,17 +123,9 @@ func cleanOpenRTBRequests(ctx context.Context,
 
 		// GDPR
 		if gdprEnforced {
-			weakVendorEnforcement := false
-			if account != nil {
-				for _, vendor := range account.GDPR.BasicEnforcementVendors {
-					if vendor == string(bidderRequest.BidderCoreName) {
-						weakVendorEnforcement = true
-						break
-					}
-				}
-			}
```

Comment:
This logic has been moved. The basic enforcement vendor check has been moved onto the `config.AccountsGDPR` object and is accessed downstream to determine weak vendor enforcement when `AuctionActivitiesAllowed` is called a few lines down from here.